src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1541,6 +1541,11 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('storage container set-permission') as c:
         c.ignore('signed_identifiers')
 
+    for item in ['set', 'show']:
+        with self.argument_context('storage container {}-permission'.format(item)) as c:
+            c.ignore('connection_string')
```

Comment:
Please keep the connection string as there's account key connection string. It's only the sas token connection string that is not supported

Reply:
Got it, will add back.